seed/src/main.rs

```diff
@@ -81,5 +81,9 @@ async fn main() {
     };
     let node = Node::new(config).unwrap();
 
-    node.run().await.unwrap();
+    let (tx, mut rx) = tokio::sync::mpsc::channel(1);
+
+    tokio::spawn(async move { while rx.recv().await.is_some() {} });
```

Comment:
:thinking:

Reply:
Yeah I mean, I guess I could log the events for now. Perhaps this `main` should be moved to `examples/`

Reply:
> moved to examples

Sounds sensible.

What's the idea eventually? Slap some prometheuses?

Reply:
Ok had a chat with xla, we're gonna get #340 merged, so this file will be removed anyway. I think `radicle-bins/seed` will serve as the example then.

Reply:
Create a fancy dashboard: https://www.figma.com/file/SQadExhzMpNYFziggQ1vDh/seed-node-ui?node-id=0%3A1 (updated pr description)